src/api/app/components/workflow_run_filter_link_component.html.haml

```diff
@@ -1,4 +1,8 @@
-= link_to(token_workflow_runs_path(@token, @filter_item), class: "list-group-item list-group-item-action #{css_for_link}") do
-  = @text
+= link_to(token_workflow_runs_path(@token, @filter_item),
+          class: "list-group-item list-group-item-action #{css_for_link} d-flex justify-content-between") do
+  %span
+    - if @icon != ''
+      %i.me-1{ class: "#{@icon} #{css_for_icon}" }
+    = @text
```

Comment:
This can be pushed into the `WorkflowRunFilterLinkComponent`

Reply:
done